01-inputs-required.tf

```diff
@@ -69,6 +69,10 @@ variable "workload_identity_enabled" {
   default = false
 }
 
+variable "aso_settings_enabled" {
```

Comment:
avoid acronyms as much as possible, this may be in context for you right now but it's unlikely anyone else would know what this means

Reply:
Updated, will update on aks-cft-deploy to reflect